docs/symboltable_serialization.md

```diff
@@ -0,0 +1,63 @@
+# Abstract
+In this document we are going to present how Jariko allows managing programs memory persistence using as approach the serialization.  
+As everyone knows the symbol table is an object containing the program memory, and generally speaking
+it is nothing more than a `Map<String, Value>` with String: the variable name, and Value: the value of it.  
+Symbol table has scope program and lives within the program, but when the program dies, if that program runs into an 
+activation group, we need to recover its state, for this reason we have thought about serialization.
```

Comment:
not "if that program runs into an activation group" but "if that program finishes with RT indicator"

Reply:
ok